pkgs/tools/misc/noti/default.nix

```
@@ -1,19 +1,23 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, Cocoa }:
+{ stdenv, lib, buildGoPackage, fetchFromGitHub
+, Cocoa ? null }:
 
-buildGoPackage rec {
-  name = "noti-${version}";
-  version = "3.1.0";
+let
+  isDarwin = stdenv.buildPlatform.isDarwin;
```

```suggestion
  isDarwin = stdenv.isDarwin;
```
I would also just inline it as well.

---

Fixed, and thanks for the review!